src/metabase/legacy_mbql/schema.cljc

```diff
@@ -1375,7 +1375,7 @@
                 (core/= tag-name (:name tag-definition)))
               m))]])
 
-(def ^:private NativeQuery:Common
+(def ^:private NativeInnerQuery:Common
```

Comment:
Renamed a couple of legacy MBQL schemas to make it clear that `MBQLQuery` refers to the so-called "inner query" and not the entire map